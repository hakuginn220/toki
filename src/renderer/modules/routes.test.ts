import reducer, { ActionTypes, initialState } from './routes'

test('CHANGE_LOCATION', () => {
  expect(
    reducer(initialState, {
      payload: { location: 'test' },
      type: ActionTypes.CHANGE_LOCATION
    })
  ).toMatchSnapshot()
})
