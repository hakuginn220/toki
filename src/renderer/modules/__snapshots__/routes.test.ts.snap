// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CHANGE_LOCATION 1`] = `
Object {
  "location": "test",
}
`;
